### ISSUE TITLE
NC13.01 - Gallery app doesn't display pictures when the News app is activated.

### ISSUE THREAD

**Issue by j-ed** [nextcloud/gallery#431]
## Expected behavior
After I've successfully migrated Nextcloud from 12.0.6 to 13.0.1 and the latest apps, I would expect that everything works as before again. Unfortunately the Gallery app seems to be negatively influenced by an installed News app.

## Current behavior
The Gallery app doesn't work as expected anymore, once the News app (**v12.0.1**)has been activated. Instead of showing a picture when you click on it, a dialog to download it or open it with an external application is shown. Additionally the following message is logged right after the News app has been activated and you're switching to the Files/Gallery app:
```
{"reqId":"jg1WtKowryXgaagjlvQm","level":3,"time":"2018-03-22 09:20:57+01:00","remoteAddr":"192.168.1.11","user":"juergen","app":"PHP","method":"GET","url":"\/index.php\/apps\/gallery\/config?extramediatypes=1","message":"Default value for parameters with a class type hint can only be NULL at \/srv\/www\/nextcloud\/nextcloud\/apps2\/news\/vendor\/symfony\/yaml\/Parser.php#49","userAgent":"Mozilla\/5.0 (Windows NT 10.0; Win64; x64; rv:59.0) Gecko\/20100101 Firefox\/59.0","version":"13.0.1.1"}
```
I've already reported this issue under the News app repository (https://github.com/nextcloud/news/issues/279) and provided trace information etc. but want to raise the awareness of this problem here too. Due the fact that the maintainer of the News app wrote **"... Doesn't really sound like a News bug to me btw. ..."** I wonder if I'm the only one hit by this problem or of someone else made a similar experience.
 
#### Server Configuration
OS: Linux 3.16.54
Web server: Apache2 2.4.29
Database: MariaDB 10.0.32
PHP version: 5.6.34
Nextcloud version: 13.0.1
News app version: 12.0.1

#### Client Configuration
Browser: Mozilla Firefox 59.0.1
Operating system: Windows 10

**Comment by j-ed** [comment]
I've upgraded Nextcloud to v13.0.2 but the described problem still persists.

**Comment by j-ed** [comment]
The problem has been solved with the latest News app version 12.0.4.

**Comment by rullzer** [comment]
Perfect!
